form_elements/styles.css

```diff
@@ -0,0 +1,159 @@
+body {
+  margin: 0;
+  background-color: hsl(0, 0%, 98%);
+  color: #333;
+  font: 100% / normal sans-serif;
+}
+
+main {
+  margin: 0 auto;
+  padding: 4rem 0;
+  width: 90%;
+  max-width: 60rem;
+}
+
+form {
+  box-sizing: border-box;
+  padding: 2rem;
+  border-radius: 1rem;
+  background-color: hsl(0, 0%, 100%);
+  border: 4px solid hsl(0, 0%, 90%);
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 2rem;
+}
+
+.full-width {
+  grid-column: span 2;
+}
+
+/* get rid of all default/special styles */
+button,
+fieldset,
+input,
+legend,
+select,
+textarea {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: transparent;
+  border: none;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* apply consistent, custom styles */
+input,
+select,
+textarea {
+  border: 2px solid #333;
+  background-color: white;
+  border-radius: 0.25rem;
+}
+
+/* any input that has a type of text */
+input[type="text"],
+/* any input that has a type of email */
+input[type="email"],
+select,
+textarea {
+  font: 1.25rem / 1.5 sans-serif;
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0.5rem 0.75rem;
+}
+
+input:focus,
+select:focus,
+textarea:focus {
+  outline: none;
+  box-shadow: 0 0 0 4px hsl(213, 90%, 70%);
+}
+
+textarea {
+  min-height: 10rem;
+}
+
+/* placeholders of any input fields, and the textarea field */
+input::placeholder,
+textarea::placeholder {
+  /* Firefox automatically reduces opacity, leading to accessibility issues. Explicitly state the opacity is 1 to fix*/
+  opacity: 1; 
+  color: hsl(213, 73%, 50%);
+}
+
+select {
+  background: url("images/down-arrow.svg") no-repeat center right 0.75rem;
```

Comment:
oo! Interesting having this as a background image 🤔 Not sure if this was a part of the tutorial or not, but it is a cool way to solve the problem of wanting a custom dropdown. I've seen people add them as `:after` elements as well!